### PR TITLE
Update Pull app graphic and readme

### PR DESCRIPTION
**Pull request by wei**
##### Checklist

- [x] If you're adding a listing for your app, be sure your app is in the `/_apps/` folder before opening this Pull Request.
- [x] All comments in frontmatter need to be removed.
- [x] Performs a useful action through the GitHub API that solves an existing problem for developers: Keep forks up-to-date with automated PRs
- [x] Is original: for example, it does something not already done by an existing Probot app
- [x] [Has tests](https://github.com/wei/pull/tree/master/test)
- [x] [Has documentation](https://github.com/wei/pull#readme)
- [x] [Is open source](https://github.com/wei/pull)
- [x] [Has a license](https://github.com/wei/pull#license)
- [x] [Has a code of conduct](https://github.com/wei/pull/blob/master/.github/CODE_OF_CONDUCT.md)
- [x] Has someone willing to at least minimally maintain them for the near future: @wei

-----
[View rendered _apps/pull.md](https://github.com/wei/probot.github.io/blob/patch-1/_apps/pull.md)

### CODE REVIEW

**Comment by wei** [comment]
### New graphic 🎉 
Pull app social graphic

**Comment by gr2m** [comment]
How many installations does pull have?

**Comment by gr2m** [comment]
I'm pretty sure it does not have as many as stale ;) The numbers on the app gallery page are unreliable because the `/probot/stats` endpoint is disabled for apps with too many installations. And the endpoint was removed for v10. We might as well remove the data from the gallery

File: _apps/pull.md [hunk 1]
```diff
@@ -25,31 +26,33 @@ organizations:
 - tiaanduplessis
 ---
 
-Keep your forks up-to-date via automatic pull requests, merges, or hard resets.
+[![Probot #1 Most Installed](https://badgen.net/badge/probot/%231%20most%20installed/orange?icon=dependabot&style=flat&cache=86400)](https://probot.github.io)
```
Comment (gr2m) [inline _apps/pull.md:29]: Could you remove this please, it's not true, I'm afraid. WIP has 7356 right now, and Stale has far, far more. I didn't check the other apps

**Comment by wei** [comment]
@gr2m I see, I ran your action last night and it came out to be 4607 installations and 108656 repositories. It took over an hour to run.

I'm using your action to update https://github.com/pull-app/stats/blob/master/stats.json and serving cached version at `/probot/stats`. My server is very happy 😄 .

**Comment by gr2m** [comment]
> Maybe I can claim most repositories?

I don't know. the latest from `stale`, but I'm sure no other github app is even close. Let me merge this in and see if I can get stats for stale, I'll let you know

**Comment by gr2m** [comment]
actually `stale` has "only" 7647 installations, not sure about repositories. Maybe we could add number of repositories as output to https://github.com/gr2m/app-stats-action?

**Comment by wei** [comment]
@gr2m great! I'll make a PR shortly.

**Comment by JasonEtco** [comment]
I'm sorry to say, but https://slack.github.com and https://lab.github.com are by far the two most installed Probot apps (with Slack being way more still). I'm not sure if I can share specifics, but I promise you they both have more installations and repositories than any other Probot app, including Stale.

**Comment by wei** [comment]
Wow didn't know they were based on probot too. That's awesome! I guess I need to add more qualifiers like the # 1 probot app that is unaffiliated with GitHub, open-source, starts with 'P', runs on a smoky server in Amsterdam, likes to brain-fart, etc 🤣

**Comment by gr2m** [comment]
Stale has 327,569 repositories right now

**Pull request merged**